src/noisepy/seis/noise_module.py

```diff
@@ -273,23 +265,23 @@ def preprocess_raw(
                 raise ValueError("The response found in the inventory is empty (no stages)! abort!")
             else:
                 try:
-                    logger.info("removing response for %s using inv" % st[0])
+                    print("removing response for %s using inv" % st[0])
                     st[0].attach_response(inv)
                     st[0].remove_response(output=rm_resp_out, pre_filt=pre_filt, water_level=60)
                 except Exception as e:
-                    logger.error("WARNING: Failed to remove response from %s. Returning empty stream. %s" % (st[0], e))
+                    print("WARNING: Failed to remove response from %s. Returning empty stream. %s" % (st[0], e))
```

Comment:
@nrich20 looks like your last commit (merge resolution) undid the replacements you had previously made. I suggest using `git revert` to undo this one.